packages/snaps-utils/src/manifest/manifest.test.ts

```diff
@@ -157,6 +164,39 @@ describe('checkManifest', () => {
     expect(version).toBe('1.0.0');
   });
 
+  it('includes new validation warnings', async () => {
```

Comment:
What new warnings are included here? It's a bit hard to tell what is being tested exactly